plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -83,9 +83,14 @@ public function __construct(SitesManager\API $api, Plugin\Manager $pluginManager
         parent::__construct($idSite, $idMeasurableType);
     }
 
+    protected function shouldShowSettingsForType($type)
+    {
+        return $type !== Type::ID;
+    }
```

Comment:
could point, I have a look. Think you're right. It still worked because there were only 2 types so didn't notice :)